src/main/java/mate/academy/taskmanagement/service/impl/DropboxServiceImpl.java

```diff
@@ -0,0 +1,120 @@
+package mate.academy.taskmanagement.service.impl;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.util.Arrays;
+import java.util.Map;
+import mate.academy.taskmanagement.exception.DropboxDeleteException;
+import mate.academy.taskmanagement.exception.DropboxDownloadException;
+import mate.academy.taskmanagement.exception.DropboxUploadException;
+import mate.academy.taskmanagement.service.DropboxService;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.io.ByteArrayResource;
+import org.springframework.core.io.Resource;
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+
+@Service
+public class DropboxServiceImpl implements DropboxService {
+    private static final String DROPBOX_UPLOAD_URL = "https://content.dropboxapi.com/2/files/upload";
+    private static final String DROPBOX_DOWNLOAD_URL = "https://content.dropboxapi.com/2/files/download";
+    private static final String DROPBOX_DELETE_URL = "https://api.dropboxapi.com/2/files/delete_v2";
+
```

Comment:
let's not hardcode such urls. Instead move them to the application.properties and reference them here using @Value annotation